cli/src/cmd/test.rs

```diff
@@ -306,29 +307,37 @@ fn test<A: ArtifactOutput + 'static>(
             }
 
             for (name, result) in tests {
-                // build up gas report
-                if gas_reporting {
-                    if let (Some(traces), Some(identified_contracts)) =
-                        (&result.traces, &result.identified_contracts)
-                    {
-                        gas_report.analyze(traces, identified_contracts);
-                    }
-                }
+                // TODO: build up gas report
+                //if gas_reporting {
+                //    if let (Some(traces), Some(identified_contracts)) =
+                //        (&result.traces, &result.identified_contracts)
+                //    {
+                //        gas_report.analyze(traces, identified_contracts);
+                //    }
+                //}
 
                 short_test_result(name, result);
 
                 // adds a linebreak only if there were any traces or logs, so that the
                 // output does not look like 1 big block.
                 let mut add_newline = false;
-                if verbosity > 1 && !result.logs.is_empty() {
+                if verbosity > 1 {
                     add_newline = true;
-                    println!("Logs:");
-                    for log in &result.logs {
-                        println!("  {}", log);
+
+                    // We only decode logs from Hardhat and DS-style console events
+                    let console_logs: Vec<String> =
+                        result.logs.iter().filter_map(|log| decode_console_log(&log)).collect();
+
+                    if !console_logs.is_empty() {
+                        println!("Logs:");
+                        for log in console_logs {
+                            println!("  {}", log);
+                        }
```

Comment:
This now collects all logs (not just Hardhat/DappTools), but I still filter them to get the behavior we had before. Does this make sense?